plugins/pyenv/pyenv.plugin.zsh

```diff
@@ -18,7 +18,10 @@ for pyenvdir in "${pyenvdirs[@]}" ; do
         export PYENV_ROOT=$pyenvdir
         export PATH=${pyenvdir}/bin:$PATH
         eval "$(pyenv init - zsh)"
-        eval "$(pyenv virtualenv-init - zsh)"
+
+        if [[ "$(pyenv commands | grep virtualenv-init -c)" == "1" ]] ; then
```

Comment:
I think this is better:
```sh
if [[ "$(pyenv commands | grep virtualenv-init -c  > /dev/null 2>&1)" == `false` ]]; then
*---* then *---*
```
If grep is succeeded, It'll see result, so I think that code is not good.